decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+# Defines the base decorator
+class Decorator
```

Comment:
- [ ] _Decorator class_ should inherit _Nameable_ like below:
```ruby
class Decorator < Nameable
    attr_accessor :nameable
  
    def initialize(nameable)

     ...

Reply:
Thank you about this. Must have been an oversight.